types/index.ts

```diff
@@ -0,0 +1,24 @@
+export type ApiResponseType<T> = {
+  count: number;
+  data: T;
+  next: string | null;
+  previous: string | null;
+};
+
+export type Post = {
```

Comment:
can we prefix the Types with a T?
also the interfaces with an I  and enums with a E?
We could also include the whole word. Just to keep it simple and quick to understand.

Reply:
Personally, it doesn't matter. We could do that if you want